<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>694</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>邮件收发管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mailmgr.qrc">
    <normaloff>:/icon/hello.jpg</normaloff>:/icon/hello.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>661</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>写信</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>931</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>发件人信息</string>
      </property>
      <widget class="QComboBox" name="smtpServer">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>smtp.163.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>smtp.126.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>smtp.yeah.net</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>smtp.qq.com</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Smtp服务器</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>邮箱账号</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="fromMail">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>100</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="domain_1">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>100</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@163.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@126.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@yeah.net</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@qq.com</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>100</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>密  码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sendPwd">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>100</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>昵  称</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sender">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>40</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>581</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>收件人信息</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>邮箱账号</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="toMail">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="domain_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>30</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@163.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@126.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@yeah.net</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@qq.com</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>昵  称</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="receiver">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>581</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>邮件信息</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>主题</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="mailSub">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="mailContent">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>491</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>内容</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addBtn">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>200</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>+添加附件</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="filePath">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>200</y>
         <width>321</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushBtn">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>240</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>190</y>
        <width>311</width>
        <height>411</height>
       </rect>
      </property>
      <property name="title">
       <string>状态信息</string>
      </property>
      <widget class="QListWidget" name="statusList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>291</width>
         <height>371</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="testBtn">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>测试</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>收信</string>
     </attribute>
     <widget class="QComboBox" name="popServer">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>pop3.163.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pop3.126.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pop3.yeah.net</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>pop3.qq.com</string>
       </property>
      </item>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>921</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>用户信息</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Pop3服务器</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="popMail">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>40</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="domain_3">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>40</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@163.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@126.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@yeah.net</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>@qq.com</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>40</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>邮箱账号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>90</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>密    码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="receivePwd">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>90</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>631</width>
        <height>441</height>
       </rect>
      </property>
      <property name="title">
       <string>邮件内容</string>
      </property>
      <widget class="QTableWidget" name="popList">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>601</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QHeaderView::section{background:rgb(85, 170, 255)};</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <column>
        <property name="text">
         <string>时间</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>发件人</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>主题</string>
        </property>
       </column>
      </widget>
      <widget class="QTextBrowser" name="popContent">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>591</width>
         <height>171</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>内容</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pullBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>接收</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>32</y>
         <width>54</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>邮件列表</string>
       </property>
      </widget>
      <widget class="QPushButton" name="extraBtn">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>390</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>@附件</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>180</y>
        <width>261</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>状态信息</string>
      </property>
      <widget class="QListWidget" name="popStatus">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>241</width>
         <height>391</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="popTest">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>0</y>
        <width>61</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>测试</string>
      </property>
     </widget>
     <zorder>groupBox_4</zorder>
     <zorder>popServer</zorder>
     <zorder>groupBox_5</zorder>
     <zorder>groupBox_7</zorder>
     <zorder>popTest</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>969</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="config">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="quit_menu">
    <property name="title">
     <string>退出</string>
    </property>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="config"/>
   <addaction name="quit_menu"/>
  </widget>
  <action name="saveFile">
   <property name="text">
    <string>保存文件</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>另存为...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mailmgr.qrc"/>
 </resources>
 <connections/>
</ui>
